1-exercises/E-array-map/exercise.js

```diff
@@ -3,11 +3,11 @@
 
 let numbers = [0.1, 0.2, 0.3, 0.4, 0.5];
 
-let numbersMultipliedByOneHundred; // complete this statement
+let numbersMultipliedByOneHundred = numbers.map((num) => num * 100);
```

Comment:
Good - this is the most concise solution. The question also asks you to write multiple solutions showing the different possible syntax.